src/ucp/api/ucp_def.h

```diff
@@ -528,8 +528,8 @@ typedef void (*ucp_tag_recv_nbx_callback_t)(void *request, ucs_status_t status,
  * @param [in]  user_data User data passed to "user_data" value,
  *                        see @ref ucp_request_param_t
  */
-typedef void (*ucp_am_data_recv_nbx_callback_t)(void *request, ucs_status_t status,
-                                                size_t length, void *user_data);
+typedef void (*ucp_am_recv_nbx_callback_t)(void *request, ucs_status_t status,
```

Comment:
let's rename to ucp_am_data_recv_nbx_callback_t to ucp_am_recv_data_nbx_callback_t
"recv_data" will be used for the "payload" while "am)recv" is for the header

Reply:
@yosefe, can you please elaborate re header?